deploy/deploy_ancillaryRH7.yml

```diff
@@ -0,0 +1,142 @@
+- hosts: wdcreancil02
+  vars:  
+    custom_repo: true
```

Comment:
@lel99999 We do not want to hard-code `custom_repo` to be true, it should only be true in remote environments.